### DISCUSSION TITLE
Any plan to support TextBundle?

### DISCUSSION THREAD

**Discussion by 45gfg9** [hexojs/hexo#5264] [Ideas]
An explanation: https://github.com/Kenshin/simpread/issues/788

TextBundle greatly simplifies the document assets organization. See [textbundle.org](https://textbundle.org).

**Comment by stevenjoezhang** [comment]
这个和 Hexo 目前的 post asset 管理很类似吧

**Reply by 45gfg9** [reply]
是的，区别就在于 Markdown 文件是与 asset 文件夹同级还是放在文件夹内。就我个人来说，我认为每篇 post 及其 assets 都集中在一个文件夹里相比现在的设计会更优雅

**Reply by stevenjoezhang** [reply]
参考 https://github.com/hexojs/hexo/issues/3245 我也同意，就是改起来很头疼

**Comment by uiolee** [comment]
sounds good. maybe it is another way to resolve the issue of local preview

**Comment by D-Sketon** [comment]
Any thoughts on adding it to v7?
